src/main/java/ledger/user_interface/ui_controllers/LoginPageController.java

```diff
@@ -0,0 +1,80 @@
+package ledger.user_interface.ui_controllers;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+import javafx.scene.Node;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+/**
+ * Created by richarel on 10/16/2016.
+ */
+public class LoginPageController extends GridPane implements Initializable {
+
+    @FXML
+    private Button chooseFileBtn;
+    @FXML
+    private TextField password;
+    @FXML
+    private Button loginBtn;
+
+    private String pwd;
+    private File file;
+
+    LoginPageController() {
+        try {
+            FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml_files/LoginPage.fxml"));
```

Comment:
Consider pulling the page locations into a static variable. Might make future changes smoother.